packages/itwinui-css/backstop/tests/tag.html

```diff
@@ -24,8 +24,11 @@
     />
     <style>
       /* Forces grouped tags to display on 1 line, can be set by the dev team. */
-      section .iui-tag-container {
-        max-width: 420px;
+      section#demo-default-container {
```

Comment:
Lets do simpler:
`#demo-default-container, #demo-basic {
  width: 440px;
}
`

Reply:
amm.. weird.. I have tried putting 440px locally, and there were no visual changes...

Reply:
Yeah, it looks the same on the html page. I don't know why the tests don't show the tag placement properly. I set the width to `445px` to fix that part. They're still changed now, but it's because the image is being taken of `#demo-default-container` and `#demo-basic`, which now have a set width of `445px` vs `fit-to-content`. The widths used to be 420px and 444px.